Formula/util-linux.rb

```diff
@@ -43,6 +43,12 @@ def install
     end
   end
 
+  def caveats
```

Comment:
This should only print on macOS iyam

Reply:
We do not have a mechanism for os-specific caveats yet. I need to write a DSL for this.